tools/debian.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# THIS SHOULD RUN ON DEBIAN FROM THE PROJECT DIRECTORY
+
+# VARIABLES
+ARCH=x64
+DEB_SIGNATURE_ID=EF506BE8
+INPUT_TYPE=dir
+LICENSE=MIT
+NODE_VERSION=8.3.0 # Will upgrade after 1.6.5
+OUTPUT_TYPE=deb
+PACKAGE_NAME=logdna-agent
+S3_BUCKET=repo.logdna.com
+VERSION=$(cat ./package.json | grep version | cut -d'"' -f4)
+
+# PREPARE FOLDER AND FILES
+mkdir -p .build/
+mv tools/shared/before-remove tools/shared/after-upgrade tools/shared/init-script tools/shared/logrotate .build/
+
+# STEP 1: INSTALL DEPENDENCIES
+# STEP 1.1: INSTALL nexe
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
```

Comment:
`-o-` looks strange.  What's that syntax mean with the dash on the end?

Reply:
@darinspivey, copied from [here](https://github.com/nvm-sh/nvm#install--update-script)